waves-integration-it/src/test/scala/com/wavesplatform/dex/grpc/integration/clients/CombinedWavesBlockchainClientTestSuite.scala

```diff
@@ -503,7 +503,7 @@ class CombinedWavesBlockchainClientTestSuite extends IntegrationSuiteBase with H
       val leasing = mkLease(bob, alice, 3.waves)
       broadcastAndAwait(leasing)
 
-      val r = Await.result(eventsF, 1.minute).foldMap(_._1.balanceUpdates)
+      val r = eventsF.futureValue.foldMap(_._1.balanceUpdates)
```

Comment:
can we preserve original timeout?

Reply:
We could do this, but there is overrided PatienceConfig with the same timeout, so it's already preserved